crates/wasmtime/src/types.rs

```diff
@@ -446,6 +462,12 @@ impl MemoryType {
     }
 }
 
+// Event types
+
+#[derive(Debug, Clone)]
+#[allow(missing_docs)]
+pub struct EventType;
```

Comment:
Since this is empty, could this perhaps be eschewed from the public API for now?